charts/reform-scan-blob-router/values.aat.template.yaml

```diff
@@ -4,6 +4,7 @@ java:
     STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 0
     DELETE_DISPATCHED_FILES_CRON: "0/10 * * * * *"
     SEND_NOTIFICATIONS_CRON: "0/10 * * * * *"
+    SEND_DAILY_REPORT_ENABLED: "false"
```

Comment:
we should not have any values here

Reply:
yes, as Don said - changes will be only in flux config